server/public/shared/i18n/i18n.go

```diff
@@ -222,3 +264,13 @@ func IdentityTfunc() TranslateFunc {
 		return translationID
 	}
 }
+
+func isSupportedLocale(locale string) bool {
+	for _, supportedLocale := range supportedLocales {
```

Comment:
0/5 since I doubt `supportedLocales` will ever be long enough for this to matter, but making it into a `map[string]bool` (ie. a pseudo-"set") would save us from having to loop over the structure

Reply:
Agreed re: looping, but since we only use this during initialization (and not, for example, on each translation attempt), I'd propose keeping the declaration of the locales simpler for now.